appinventor/aicompanionapp/src/ViewController.swift

```diff
@@ -139,8 +139,22 @@ public class ViewController: UINavigationController, UITextFieldDelegate {
     if let menuButton = viewControllers.last?.navigationItem.rightBarButtonItem {
       menuButton.action = #selector(openMenu(caller:))
       menuButton.target = self
+      if #available(iOS 13.0, *) {
+        menuButton.image = UIImage(systemName: "book")
+      }
```

Comment:
I'm not sure this is intentional here. If the ViewController is going to appear it's also the last item, and this causes the icon to reset to the book after reviewing the onboarding screens rather than staying a ?. We may need to look at something more specific. In #3467 there is a flag added for this purpose so we may want to get that merged first and make use of it here.

Reply:
if the default setting for the Main.storyboard is the `questionmark.circle`, this code would only be necessary if there is any interchange with forms showing and not showing. We would want the "book" to show once a project is launched, and then to flip back to "questionmark" if all projects are closed and we land on the main screen again

Reply:
the larger conditional is ``` if let menuButton = viewControllers.last?.navigationItem.rightBarButtonItem {
      menuButton.action = #selector(openMenu(caller:))
      menuButton.target = self
      if #available(iOS 13.0, *) {
        menuButton.image = UIImage(systemName: "book") // necessary if we toggle back?
      }
    }
    if (form == nil) {
      
      if let menuButton = viewControllers.last?.navigationItem.rightBarButtonItem {
        menuButton.action = #selector(goBackToOnboarding(caller:))

        if #available(iOS 13.0, *) {
          menuButton.image = UIImage(systemName: "questionmark.circle")
        } else {
          menuButton.title = "?" // redundant, can remove
        }
        menuButton.title = "?"
      }```